ngi_reports/stockholm/project_summary.py

```diff
@@ -77,29 +84,49 @@ def __init__(self, config, LOG, working_dir, **kwargs):
         self.project_info['accredit'] = self.get_accredit_info(['library_preparation','sequencing','data_processing','data_analysis'])
-
-        ## Collect information about the sample preps
+        self.project_info['total_lanes'] = 0
+        self.project_info['display_limit'] = kwargs.get('display_limit')
+        self.project_info['missing_fc'] = False
+        self.project_info['aborted_samples'] = {}
+        
+        ## Collect information about the sample preps and collect aborted samples
         for sample_id, sample in sorted(self.proj.get('samples', {}).iteritems()):
             self.LOG.info('Processing sample {}'.format(sample_id))
+            ## Check if the sample is aborted before processing
+            if sample.get('details',{}).get('status_(manual)') == "Aborted":
+                self.LOG.info('Sample {} is aborted, so skipping it'.format(sample_id))
+                self.project_info['aborted_samples'][sample_id] = {'user_id': sample.get('customer_name',''), 'status':'Aborted'}
+                continue
+            
             ## Basic fields from Project database
             self.samples_info[sample_id] = {'ngi_id': sample_id}
             self.samples_info[sample_id]['customer_name'] = sample.get('customer_name','')
-            self.samples_info[sample_id]['total_reads'] = sample.get('details',{}).get('total_reads_(m)','')
-            if self.project_info['ordered_reads']:
-                self.samples_info[sample_id]['seq_status'] = ['FAIL','PASS'][float(self.samples_info[sample_id]['total_reads']) > \
-                                                                         float(self.project_info['ordered_reads'].replace('M',''))]
-            else:
-                self.samples_info[sample_id]['seq_status'] = None
+            self.samples_info[sample_id]['total_reads'] = sample.get('details',{}).get('total_reads_(m)')
+            self.samples_info[sample_id]['reads_min'] = sample.get('details',{}).get('reads_min')
+            
+            ## Check if a non-aborted sample is sequenced
+            if self.samples_info[sample_id]['total_reads'] == None:
+                self.LOG.warn("Sample {} dont have total reads, so adding it to NOT sequenced samples list.".format(sample_id))
+                self.project_info['aborted_samples'][sample_id] = {'user_id': sample.get('customer_name',''), 'status':'Not sequenced'}
+                del self.samples_info[sample_id]
+                continue
+            
+            ## Check if reads minimum set for sample and the status if it does
+            if self.samples_info[sample_id]['reads_min']:
+                self.project_info['ordered_reads'].append("{}M".format(self.samples_info[sample_id]['reads_min']))
+                if float(self.samples_info[sample_id]['total_reads']) > float(self.samples_info[sample_id]['reads_min']):
+                    self.samples_info[sample_id]['seq_status'] = 'PASS'
```

Comment:
Is this used for anything other than printing in the report? If not, could we have `Passed` and `Failed` instead of `PASS` and `FAIL`? Just seems a bit more polite :gb: